onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorAttention.cpp

```diff
@@ -49,397 +49,501 @@ class DmlOperatorAttention : public DmlOperator
     DmlOperatorAttention(const MLOperatorKernelCreationContext& kernelCreationContext)
     :   DmlOperator(kernelCreationContext)
     {
-        ML_CHECK_VALID_ARGUMENT(kernelCreationContext.GetInputCount() >= 3);
+        enum DmlInputIndex : uint32_t
+        {
```

Comment:
[nit]
I know we want to push this change as soon as possible. 
If possible, can we please add ASCII graph for these 2 implementations or a snapshot of any other graph as part of the PR? It will help in reviewing this PR and in-future too.

Reply:
Sure, I'll update the `Attention` graph. `MultiHeadAttention` doesn't have a graph so it shouldn't be needed.

Reply:
Mmm, ASCII art. 😊